11-parallelism/src/tests.rs

```diff
@@ -0,0 +1,59 @@
+#![allow(unused_mut, unused_variables, dead_code)]
+
+mod simple_thread {
+
+    use std::thread;
+
+    #[test]
+    fn spawn_a_thread_and_get_his_id() {
+        let father_id = thread::current().id();
+        let mut child_id = thread::current().id(); // The goal is to change that value
```

Comment:
I suggest to remove `mut` and just declare child_id after second TODO

Reply:
I put this one to have an assert and the test can compile. I prefer when tests can compile.

Reply:
Ok, then just remove `mut` and move `child_id` to the right place

Reply:
Done